### ISSUE TITLE
temporary entities in NGSI-LD

### ISSUE THREAD

**Issue by FR-ADDIX** [Fiware/context.Orion-LD#1300]
Is there an equivalent in NGSI-LD to the dateExpires under NGSI V2 with which you can delete entities from the ContextBroker at a specified time?

**Comment by kzangeli** [comment]
Nope, that feature isn't part of NGSI-LD.
Do you find it useful?

**Comment by FR-ADDIX** [comment]
Yes I find this very helpful, so you can create Temporary Entities that disappear at a predetermined time. Departure schedules at bus stops, for example, are based on such a mechanism. Also, if we observe ships in an area, for example, without the mechanism more and more entities would appear in the area, even if the signal has not been received for a long time. In the ContextBroker we are only interested in the current status of the ships and after 24 hours without signal the vehicle can disappear. In the historical data we still have it with its route but in the CB it is no longer important.
Without this functionality we have to constantly check the CB for expired entities and then delete them ourselves. with an expresAt the Orion does this for us.

**Comment by kzangeli** [comment]
Ok, I'll propose this feature for ngsi-ld in etsi isg cim.
I believe Orion calls the feature "transient" entities

**Comment by agaldemas-eridanis** [comment]
Hello, I confirm the great usefulness of this feature in many cases, we already used it in several NGSI-V2 project, then it's really valuable to add it !

**Comment by kzangeli** [comment]
ok, duly noted
